src/server.js

```diff
@@ -215,5 +224,76 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_ADDRESS}/geocode`,
+  async (request, response) => {
+    let address = request.params[URL_PARAM_ADDRESS];
+
+    let geocodeRequest =
+      "https://maps.googleapis.com/maps/api/geocode/json?address=" +
+      address +
+      "&key=" +
+      env.API_KEY;
+
+    const geocodeResponse = await (await fetch(geocodeRequest)).json();
+
+    try {
+      response.json({
+        status: 200,
+        data: geocodeResponse.results[0].geometry.location,
+      });
+    } catch (err) {
+      console.error(err);
+      response
+        .status(500)
```

Comment:
I wonder if it would make sense to pass through the error code from the geocode request itself here?

Reply:
I was just updating this. I saw Ved's comment on Chisom's PR.

I now put the api within the try block and check for a status of "OK". If an error gets caught, or the geocoding api response status is not "OK", I return that error to the client.